arch/arc/Kconfig

```diff
@@ -337,6 +337,23 @@ config ARC_NORMAL_FIRMWARE
 	  resources of the ARC processors, and, therefore, it shall avoid
 	  accessing them.
 
+config ARC_HAS_DSP
+	bool
+	help
+	  This option is enabled when the ARC CPU has hardware DSP unit.
+
+menu "ARC DSP Options"
+depends on ARC_HAS_DSP
+
+config ARC_DSP
+	bool "This option enables digital signal processing (DSP)"
+	help
+	  This option enables DSP and DSP instructions.
```

Comment:
Let's move it also to `arch/arc/core/dsp/Kconfig` - so we only have `source "arch/arc/core/dsp/Kconfig"` here.